Sources/Commands/Workspace.swift

```diff
@@ -269,6 +271,71 @@ public class Workspace {
         return path
     }
 
+    /// Updates the current dependencies.
+    public func updateDependencies() throws {
+        let (rootManifest, _) = try loadDependencyManifests()
+        // Only create constraints based on root manifest for the update resolution.
+        let updateConstraints = computePackageConstraints(rootManifest: rootManifest, dependencies: [])
+        // Resolve the dependencies.
+        let updateResults = try resolveDependencies(constraints: updateConstraints)
+
+        for (specifier, version) in updateResults {
```

Comment:
I think that's likely to provable valuable down the line, but maybe not worth doing right now? For the current command line usage I think I would rather have a model where we remember the old version state to let you easily roll back rather than a model where we prompt and you approve.

It should be easy to refactor to add this if we need want it later...